### PR TITLE
Remove some obsolete protocols

### PR DESCRIPTION
**Pull request by 41CEA55**
Please sign (check) the below before submitting the Pull Request:

- [x] I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
- [x] I have read the contributing guide lines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
- [x] I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Describe changes:

Spanish social network Tuenti closed in 2016, telephony brand in 2022, their website is down.

https://english.elpais.com/elpais/2016/02/02/inenglish/1454408882_689469.html
https://www.isitdownrightnow.com/tuenti.com.html
https://internetrepublic.com/the-long-goodbye-of-the-social-network-tuenti/
https://www.telecompaper.com/news/telefonica-confirms-closure-of-tuenti-brand--1422668

TVU Networks discontinued service for their TVU Player at the end of February 2013, so this protocol is long dead.

https://www.ghacks.net/2013/03/11/with-tvu-player-gone-what-are-some-working-pc-tv-alternatives/

There were also rumors of Kontiki closing in 2011 after being acquired by Verisign, but so I haven't found any proof of that yet.

### CODE REVIEW

File: tests/cfgs/default/result/sites.pcapng.out [hunk 1]
```diff
@@ -111,8 +110,8 @@ JA3 Host Stats:
...
+	26	TCP 192.168.1.128:51248 <-> 95.131.169.91:443 [proto: 91/TLS][IP: 0/Unknown][Encrypted][Confidence: DPI][DPI packets: 6][cat: Web/5][3 pkts/849 bytes <-> 5 pkts/3703 bytes][Goodput ratio: 76/91][0.14 sec][Hostname/SNI: tuenti.com][(Advertised) ALPNs: h2;http/1.1][TLS Supported Versions: TLSv1.3;TLSv1.2][bytes ratio: -0.627 (Download)][IAT c2s/s2c min/avg/max/stddev: 46/0 47/24 48/47 1/24][Pkt Len c2s/s2c min/avg/max/stddev: 74/74 283/741 583/1514 218/647][TLSv1.2][JA3C: 579ccef312d18482fc42e2b822ca2430][JA4: t13d1715h2_5b57614c22b0_3d5424432f57][ServerNames: *.tuenti.com,tuenti.com][JA3S: 61be9ce3d068c08ff99a857f62352f9d][Issuer: C=US, O=DigiCert Inc, CN=DigiCert TLS RSA SHA256 2020 CA1][Subject: C=ES, L=Madrid, O=Tuenti Technologies S.L., CN=*.tuenti.com][Certificate SHA-1: 89:B8:FA:C7:22:04:D2:BE:C5:6E:59:10:31:67:42:B1:3F:6D:F8:3B][Firefox][Validity: 2021-04-05 00:00:00 - 2022-05-06 23:59:59][Cipher: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256][Plen Bins: 0,16,0,16,0,0,0,0,0,0,0,0,0,16,0,0,16,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,34,0,0]
```
Comment (IvanNardi) [inline tests/cfgs/default/result/sites.pcapng.out]: Could you try to remove these 2 flows from the trace, please?

File: src/include/ndpi_protocol_ids.h [hunk 2]
```diff
@@ -87,7 +87,7 @@ typedef enum {
   NDPI_PROTOCOL_YANDEX_MARKET         = 56,
   NDPI_PROTOCOL_YANDEX_DISK           = 57,
   NDPI_PROTOCOL_DISCORD               = 58,
-  NDPI_PROTOCOL_TVUPLAYER             = 59,
+  NDPI_PROTOCOL_FREE_59               = 59,
```
Comment (IvanNardi) [inline src/include/ndpi_protocol_ids.h]: "/* FREE */ " comment...

File: tests/cfgs/default/result/sites.pcapng.out (hunk 1 above)
Comment (41CEA55) [reply tests/cfgs/default/result/sites.pcapng.out:114]: > Could you try to remove these 2 flows from the trace, please?

Yea, sure.

File: src/include/ndpi_protocol_ids.h (hunk 2 above)
Comment (41CEA55) [reply src/include/ndpi_protocol_ids.h:90]: > "/* FREE */ " comment...

Sorry, I missed it.

**Review by IvanNardi — APPROVED** [review]
I am always happy when old stuff is removed :)

**Comment by 41CEA55** [comment]
> I am always happy when old stuff is removed :)

So, Kontiki is next? I looked for news about its closure, but all the links were broken and nothing was saved in WebArchive. Also there's no mentions about it after 2010.

**Comment by IvanNardi** [comment]
> > I am always happy when old stuff is removed :)
> 
> So, Kontiki is next? I looked for news about its closure, but all the links were broken and nothing was saved in WebArchive. Also there's no mentions about it after 2010.

Yes, we can remove it, too

**Review by IvanNardi — APPROVED** [review]

**Comment by IvanNardi** [comment]
Thanks. I slightly updated the commit message during the merge

**Pull request merged**